src/utilities/toast.ts

```diff
@@ -16,6 +16,11 @@ function findToast(toastText: string, checkIsClosed: boolean = false) {
 }
 
 function hideToast(toast: Toast) {
+  // Only hide toast if in browser context
+  if (typeof document === 'undefined') {
```

Comment:
There is the `browser` boolean from `$app/environment` that can be used if you want to be more "svelte-like". I don't feel too strongly one way or another, so feel free to ignore. I will say that I've had instances where doing checks on `document` or `window` wasn't enough for me before. Maybe I wasn't doing it correctly or something. So the svelte variable might be a nice dependable method of checking. Ok so maybe I'm leaning more towards the svelte way, but it's your call.

Reply:
Ohh. Ugh. Yeah I guess none of the code that uses the `browser` variable have unit tests so this is probably the first time we've encountered it...

Reply:
@duranb any more thoughts on this one? I think _for now_ maybe we should punt on getting `browser` to work in Vitest, see https://github.com/vitest-dev/vitest/issues/2834 for a long and somewhat related discussion on this, would likely require adding in some weird workaround in test config for this single test?